packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/layout_explorer_widget.dart

```diff
@@ -183,8 +183,8 @@ abstract class LayoutExplorerWidgetState<W extends LayoutExplorerWidget,
   // this is required so that we don't change focus
   //   when tapping on a child is also Flex-based widget.
   Future<void> setSelectionInspector(RemoteDiagnosticsNode node) async {
-    final service = node.inspectorService;
-    if (service is ObjectGroup) {
+    final service = node.objectGroupApi;
+    if (service != null && service is ObjectGroup) {
```

Comment:
this null check should be redundant. `service is ObjectGroup` should return false if service is null

Reply:
For some reason `service is ObjectGroup` does not give non-nullness that I need in this code block.